### PR TITLE
LPS-174275 Improving the testflow creation header

### PR DESCRIPTION
**Pull request by caiosouza15**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildOutlet.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ const BuildOutlet: React.FC<BuildOutletProps> = ({ignorePaths}) => {
 				},
 			]);
 		}
-	}, [setHeading, testrayProject, testrayRoutine, testrayBuild]);
+	}, [setHeading, testrayProject, testrayRoutine, testrayBuild, pathname]);
```
Comment (Elvinho) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildOutlet.tsx:112]: @caiosouza15 can you order it?

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildOverview.tsx [hunk 2]
```diff
@@ -45,9 +45,7 @@ const BuildOverview: React.FC<BuildOverviewProps> = ({
 			{!testrayTask && (
 				<ClayButton
 					className="mb-4"
-					onClick={() =>
-						navigate(`/testflow/${testrayBuild.id}/create`)
-					}
+					onClick={() => navigate(`testflow/create`)}
```
Comment (Elvinho) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildOverview.tsx:48]: usar aspas simples, no lugar de template String

**Review by Elvinho — CHANGES_REQUESTED** [review]
Good job @caiosouza15 , I left few suggestions for you

**Comment by kevenleone** [comment]
ci:forward

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

**Pull request closed without merging**